src/main/kotlin/com/jay/racingcar/domain/RacingCars.kt

```diff
@@ -0,0 +1,17 @@
+package com.jay.racingcar.domain
+
+class RacingCars(val racingCars: List<RacingCar>) {
```

Comment:
```suggestion
class RacingCars private constructor(val racingCars: List<RacingCar>) {
```
create라는 Factory 함수를 잘 만들어주셨네요. 👍 
그렇다면 외부에서 객체 생성 시 create 함수에만 접근할 수 있도록, 생성자를 private하게 만들어보면 어떨까요?

Reply:
와우 감사합니다~